lib/install/management/delete.go

```diff
@@ -288,6 +311,7 @@ func (d *Dispatcher) DeleteVCHInstances(vmm *vm.VirtualMachine, conf *config.Vir
 				errs = append(errs, err.Error())
 				mu.Unlock()
 			}
+			d.op.Debugf("successfully deleted cvm: %s", child.InventoryPath)
```

Comment:
Also minor: s/successfully/Successfully to maintain consistent casing for vic-machine log messages.